schedule/yam/agama_gnome_ppc64le.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_gnome_ppc64le
```

Comment:
Please comments for the reason we remove first_boot on ppc64le.

Reply:
> Please comments for the reason we remove first_boot on ppc64le.

Added.